#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

Header header
float32 x_primary
float32 y_primary
float32 z_primary
float32 heading
float32 pitch
float32 distance
float32 x_primary_accuracy
float32 y_primary_accuracy
float32 z_primary_accuracy
float32 heading_accuracy
float32 pitch_accuracy
float32 distance_accuracy